tap_github/__init__.py

```diff
@@ -1836,19 +1836,15 @@ def get_all_commits(schema, repo_path,  state, mdata, start_date):
     https://developer.github.com/v3/repos/commits/#list-commits-on-a-repository
     '''
 
-    bookmark = get_bookmark(state, repo_path, "commits", "since", start_date)
-    if not bookmark:
-        bookmark = '1970-01-01'
-
     # Get the set of all commits we have fetched previously
     fetchedCommits = get_bookmark(state, repo_path, "commits", "fetchedCommits")
     if not fetchedCommits:
         fetchedCommits = {}
-    else:
-        # We have run previously, so we don't want to use the time-based bookmark becuase it could
-        # skip commits that are pushed after they are committed. So, reset the 'since' bookmark back
-        # to the beginning of time and rely solely on the fetchedCommits bookmark.
-        bookmark = '1970-01-01'
+
+    # We have run previously, so we don't want to use the time-based bookmark becuase it could
```

Comment:
Can you please update this comment too and say that there's never a case where we don't want to keep going back through all commits until we have everything that's referenced?

Reply:
And the same comment below...